src/shared/templates/readmes/basic-heading.md

```diff
@@ -5,7 +5,7 @@
 <p align="center">
 {{ #noTagLine }}{{ /noTagLine }}
 {{ ^noTagLine }}
-✨ A beautiful dark theme for {{ portName }} and another apps
+✨ A beautiful dark theme for {{ portName }} and other apps.
```

Comment:
Hi, could you please remove the dot at the end of all tagline changes? Just `✨ A beautiful dark theme for {{ portName }} and other apps`

Reply:
done :+1: